### ISSUE TITLE
%r causes W: Ambiguous regexp literal rule to be ignored

### ISSUE THREAD

**Issue by vinyar** [bbatsov/rubocop#1922]
__Example failing test:__
```ruby
  describe command('sudo apt-get update') do
    its(:exit_status) { should eq 0 }
    its(:stdout) { should match(/Ign https:\/\/get.docker.com docker\/main/) }
  end
C: Use %r for regular expressions matching more than 1 '/' character.
```

__Example passing tests:__
```ruby
  describe command('sudo apt-get update') do
    its(:exit_status) { should eq 0 }
    its(:stdout) { should match %r{Ign https://get.docker.com docker/main/} }
    its(:stdout) { should match(%r{/Ign https://get.docker.com docker/main/}) }
  end
```
__Expectation:__ below example should fail:
```ruby
    its(:stdout) { should match %r{Ign https://get.docker.com docker/main/} }
```
Expected error:
```ruby
 W: Ambiguous regexp literal.
Parenthesize the method arguments if it's surely a regexp literal, or add a 
whitespace to the right of the / if it should be a division.
```

**Comment by bbatsov** [comment]
Well, this is no longer ambiguous - `/` can be treated as division, but `%r` always marks the start of a regexp literal. Or am I missing something in your report?

**Comment by vinyar** [comment]
I figured     its(:stdout) { should match %r{...} }  should fail because %r is not wrapped in parentheses the way other expressions are expected to be.

**Comment by alexdowad** [comment]
This is a non-issue because `%r{...}` is not an ambiguous regexp literal under any circumstances. It should be closed.

**Comment by vinyar** [comment]
I defer to experts to close as they see fit
